source/common/http/header_map_impl.h

```diff
@@ -80,6 +81,9 @@ class HeaderMapImpl : public HeaderMap {
   size_t size() const override { return headers_.size(); }
 
 protected:
+  void copyFrom(const HeaderMap& rhs);
```

Comment:
comment that these are protected because they are suspected to be slow, and should only be used in tests?